Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:8.9
+#note docker node image comes with yarn
+
+WORKDIR /agriculture-core/
+
+#ADD . /agriculture-core/
```

Comment:
The `ADD` command will also lock the changes to the image, the image will need to be reinitialized every time changes to the repo is made. So instead of building this with the ADD command, we should specify how to start the container with the `--volume` option.

Reply:
I think the issue is that I need to build the code in the container during the image build process. That's why the VOLUME command won't work. it doesn't matter if we reference it at run time because we need the code at the time we build the image so we can build the code within the container. The only way around this would be to find a way to build it when we run.